Sources/Card/CardClient.swift

```diff
@@ -0,0 +1,52 @@
+import Foundation
+import PaymentsCore
+
+public class CardClient {
+
+    let apiClient: APIClient
+    let config: CoreConfig
+
+    /// Initialize a CardClient to process card payment
+    /// - Parameter config: The CoreConfig object
+    public init(config: CoreConfig) {
+        self.config = config
+        self.apiClient = APIClient(environment: config.environment)
+    }
+
+    /// For internal use for testing/mocking purpose
+    init(config: CoreConfig, apiClient: APIClient) {
```

Comment:
```suggestion
    internal init(config: CoreConfig, apiClient: APIClient) {
```

Reply:
I believe `internal` is by default

Reply:
Swift defaults to internal if you don't specify an access control level. I think typically I'd lean toward not specifying internal but if we feel like it's necessary and provides clarity, happy to be overruled.

Reply:
Just saw Minh made the same comment 4 hours ago 🤦‍♂️  probably should have refreshed the PR